lib/matplotlib/contour.py

```diff
@@ -1406,7 +1406,12 @@ def find_nearest_contour(self, x, y, indices=None, pixel=True):
                     ymin = xc[1]
 
         return (conmin, segmin, imin, xmin, ymin, dmin)
-
+  def remove(self):
```

Comment:
why is this needed?

Reply:
Because it does not have this method and it is rather annoying to sort out how to remove contours objects.

Reply:
To avoid reploting Basemap when we want to remove contours objects and plot a new one.